docs/src/Parser/FunctionCallReflector.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Class FunctionCallReflector.
+ *
+ * @package AmpProject\AmpWP
+ */
+
+namespace AmpProject\AmpWP\Documentation\Parser;
+
+use phpDocumentor\Reflection\BaseReflector;
+use PhpParser\Node\Expr\ArrayDimFetch;
+use PhpParser\Node\Expr\Variable;
+use PhpParser\Node\Name;
+use PhpParser\Node\Name\FullyQualified;
+
+/**
+ * A reflection of a function call expression.
+ */
+final class FunctionCallReflector extends BaseReflector {
+
+	/**
+	 * Returns the name for this Reflector instance.
+	 *
+	 * @return string
+	 */
+	public function getName() {
+		if ( isset( $this->node->namespacedName ) ) {
+			return '\\' . implode( '\\', $this->node->namespacedName->parts );
```

Comment:
Is this expected?

![image](https://user-images.githubusercontent.com/134745/91347224-d1d92100-e796-11ea-8de6-5e91ba65a355.png)